contextual-identities/README.md

```diff
@@ -0,0 +1,11 @@
+# Contextual Identities
+
+## What it does
+
+Lists existing idenities, lets you create new tabs with an identity and remove all tabs from an identity.
```

Comment:
Maybe it's also worth linking to https://wiki.mozilla.org/Security/Contextual_Identity_Project/Containers for context?